packages/ui/src/components/modals/ConditionModal.tsx

```diff
@@ -149,19 +240,21 @@ const Component: React.FC<Props> = (props) => {
             setEnumValueOptions(valueOptions)
             // Only allow equal operator when selecting enum
             setSelectedOperatorOption(equalOperatorOption)
-            setShowNumberValue(false)
-        }
-        else {
-            setShowNumberValue(true)
+            setShowStringField(false)
+        } else if (!entity.isMultivalue) {
+            setSelectedOperatorOption(stringEqualOperatorOption)
+            setShowStringField(true)
+            console.log(selectedComparisonType)
+        } else { // multivalue
```

Comment:
Prefer not to have inline comments. Would move to line above. Makes text manipulation and movement easier

Like:
```
...
}
// comment
else {
  ...
}
```

Reply:
sure, ok done.